templates/cli/lib/utils.js.twig

```diff
@@ -14,6 +14,16 @@ function getAllFiles(folder) {
     return files;
 }
 
+const checkDeployConditions = (entityList, localConfig, entityType) => {
+    if (entityList.length === 0 && !localConfig.configFileAbsent) {
+        throw new Error(`No ${entityType}s found in the current directory.`);
+    }
```

Comment:
I don't think we need this because there's already a check for this because the following line already does this, right?